packages/bot-web-ui/src/components/dashboard/tutorial-components/tutorial.scss

```diff
@@ -0,0 +1,129 @@
+            }
+        }
+
+        .dc-tabs {
+            &__wrapper {
+                width: 85%;
+            }
+
+            &--top {
+                display: flex;
+                height: calc(100vh - 16rem);
+            }
+
+            &__list {
+                width: 22.5rem;
+                display: flex;
+                flex-direction: column;
+                height: calc(100vh - 16rem);
+            }
+
+            &__active {
+                width: 100%;
+            }
+
+            &__item {
+                width: 100%;
+                border-radius: 0;
+
+                &:first-child {
+                    margin-top: 6rem;
+                }
+            }
+
+            &__inner-content {
+                width: 100%;
+                padding: 1.6rem 2.4rem;
+
+                h1 {
+                    font-size: 1.6rem;
+                    font-weight: 700;
+                    margin-bottom: 2.4rem;
+                }
+
+                &--card {
+                    display: flex;
+                    text-align: center;
+
+                    &-holder {
+                        margin-right: 2.4rem;
+                        height: 16rem;
+                        width: 28rem;
+
+                        &-placeholder {
+                            display: flex;
+                            justify-content: center;
+                            align-items: center;
+                            height: 100%;
+                            width: 100%;
+                            background: #d6d6d6;
+                            margin-bottom: 0.8rem;
+
+                            &--button-group {
+                                display: flex;
+                                justify-content: center;
+                                align-items: center;
+                                width: 20rem;
+                                height: 7rem;
+                                border-radius: 10px;
+                                background-color: rgba(0, 0, 0, 0.5);
+                                &-play {
+                                    cursor: pointer;
+                                    filter: invert(1);
+                                }
+                            }
+                        }
+
+                        &-label {
+                            text-align: center;
+                            font-size: 1.4rem;
+                            font-weight: 400;
+                        }
+                    }
+                }
+            }
+        }
+
+        .dc-accordion {
+            &__item-header {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+            }
+            &__item {
+                border: unset;
+                border-bottom: 1px solid var(--general-section-1);
+            }
+        }
+
+        &__faq_header {
+            margin-bottom: 1rem;
+            margin-left: 0.5rem;
```

Comment:
can we write in a shorter way? 
`margin: 0 0 1rem 0.5rem`

Reply:
yes sure